catalog/core/catalog-core-impl/filter-proxy/src/main/java/ddf/catalog/filter/proxy/adapter/GeotoolsFilterAdapterImpl.java

```diff
@@ -103,6 +107,9 @@
 
     private static final FilterFactory FF = new FilterFactoryImpl();
 
+    private static final Pattern RELATIVE_TEMPORAL_MATCHER = Pattern.compile(
+            "RELATIVE\\(P(?!$)(\\d+Y)?(\\d+M)?(\\d+W)?(\\d+D)?(T(?=\\d)(\\d+H)?(\\d+M)?(\\d+S)?)?\\)");
```

Comment:
We are not technically supporting the smallest component being fractional, but looking into if there is another way to validate instead of obfuscating the regex.. At a minimum we should probably just leave a little note stating that it does not cover that.